ApplicationPattern.yaml

```diff
@@ -8087,6 +8191,270 @@ paths:
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredOamRequests'
+  /core-model-1-4:control-construct/profile-collection/profile={uuid}/response-profile-1-0:response-profile-pac/response-profile-configuration/value:
+    parameters:
+      - in: path
+        name: uuid
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-response-p-([0-9]{3})$'
+          example: 'ro-2-0-0-response-p-000'
+    get:
+      operationId: getGenericResponseProfileValue
+      summary: 'Returns the Value of the Field'
+      tags:
+        - GenericResponseProfile
+      security:
+        - basicAuth: []
+      responses:
+        '200':
+          description: 'Value of the field provided'
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - response-profile-1-0:value
+                properties:
+                  response-profile-1-0:value:
+                    type: object
+                    minProperties: 1
+                    maxProperties: 1
+                    properties:
+                      concrete-value:
+                        type: string
+                      value-reference:
+                        type: string
+                    example: 
+                      value-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/application-name'
+        '400':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '401':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '403':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredOamRequests'
+    put:
+      operationId: putGenericResponseProfileValue
+      summary: 'Configures the Value of the Field'
+      tags:
+        - GenericResponseProfile
+      security:
+        - basicAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - response-profile-1-0:value
+              properties:
+                response-profile-1-0:value:
+                  type: object
+                  minProperties: 1
+                  maxProperties: 1
```

Comment:
Similar to above comment, adding "additionalProperties: false" restriction would help in this situation to restrict entry of incorrect property in request-body

Reply:
Changed as proposed despite it is useless at a GET.